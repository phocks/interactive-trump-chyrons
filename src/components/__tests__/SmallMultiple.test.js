const React = require('react');
const renderer = require('react-test-renderer');

const SmallMultiple = require('../SmallMultiple');

describe('SmallMultiple', () => {
  test('It renders', () => {
    const component = renderer.create(<SmallMultiple />);
    
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
