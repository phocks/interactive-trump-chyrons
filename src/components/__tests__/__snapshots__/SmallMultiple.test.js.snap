// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SmallMultiple It renders 1`] = `
<div
  className="wrapper "
/>
`;
